Formula/php-spx.rb

```diff
@@ -0,0 +1,19 @@
+require_relative "../lib/php_pecl_formula"
+
+class PhpSpx < PhpPeclFormula
+  extension_dsl "SPX - A simple profiler for PHP"
```

Comment:
Use the [upstream supplied](https://github.com/NoiseByNorthwest/php-spx/blob/v0.4.20/composer.json#L5) values.

```suggestion
  extension_dsl "A simple & straight-to-the-point PHP profiling extension with its built-in web UI"
```